tubular/mixins.py

```diff
@@ -60,6 +60,30 @@ def drop_original_column(
         return X
 
 
+class NewColumnNameMixin:
+    """Helper to validate and set new_column_name attribute"""
+
+    def check_and_set_new_column_name(self, new_column_name: str) -> None:
+        if not (isinstance(new_column_name, str)):
+            msg = f"{self.classname()}: new_column_name should be str"
+            raise TypeError(msg)
+
+        self.new_column_name = new_column_name
+
+
+class TwoColumnMixin:
+    """helper to validate columns when exactly two columns are required"""
+
+    def check_two_columns(self, columns: list[str] | str) -> None:
```

Comment:
Q: Can two column input be just a single str? Would that not fail the if not (isinstance(columns, list)) test

Reply:
Thanks David